docs/labels.md

```diff
@@ -7,7 +7,11 @@ There are two main types of labels that kops can create:
 
 Both are specified at the InstanceGroup level.
 
-A nice use for CloudLabels is to specify [AWS cost allocation tags](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html)
+A nice use for CloudLabels is to specify [AWS cost allocation tags](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html).
+
+A good use for nodeLables is to implement [nodeSelector labels](https://kubernetes
+.io/docs/concepts/configuration/assign-pod-node/#step-two-add-a-nodeselector-field-to-your-pod-configuration) that 
+survive [AWS EC2 auto scaling groups](https://aws.amazon.com/autoscaling/) replacing unhealthy or unreachable instances.
```

Comment:
I would probably remove "that survive replacing unhealthy or unreachable instances", because cloudLabels do too?  Or was the intention to contrast with manually adding the labels to the nodes?

I like the messaging, but I think it's probably clearer with _just_ "A good use for nodeLables is to implement [nodeSelector labels]"